app/javascript/components/widgets/twitter/tweet_content.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Highlighter from 'react-highlight-words';
+import Microlink from '@microlink/react';
+import { colors, fonts, fontSizes } from '../../../lib/theme';
+
+const ContentWrapper = styled.div`
+  display: block;
+  width: auto;
+  height: 487px;
+  margin-bottom: 29px;
+`;
+
+const ContentImage = styled.img`
+  display: block;
+  width: auto;
+  height: 487px;
+  border-radius: 6px;
+  border-collapse: separate;
+`;
+
+const TweetText = styled.div`
+  color: #959292;
+  font-size: ${fontSizes.large};
+  margin-bottom: 29px;
+`;
+
+const getMentionsAndTags = text => {
+  const allWords = text.split(/[\s,\n]+/);
+  return allWords.filter(word => word.startsWith('@') || word.startsWith('#'));
```

Comment:
Some repetition with this function. Better to extract it into a utils.js in the folder and share.